qml/ConnectionMenu.qml

```diff
@@ -197,9 +203,29 @@ Item {
                     text: "1234"
                     Layout.columnSpan:  2
                     Layout.fillWidth: true
-                    onEditingFinished: {
-                        connect(AbstractLinkNamespace.Udp, udpIp.text, text)
```

Comment:
unrelated to the patch, but 'connect' is a very bad name for anything that deals with Qt.

Reply:
Function removed.